test/Performance/report.sql

```diff
@@ -0,0 +1,24 @@
+--The script can be run to read the results

```

Comment:
add ". You can filter out the results ran on a certain date by changing the date at the end."